test/boilerplate_suggestors/simple_props_and_state_class_migrator_test.dart

```diff
@@ -12,25 +12,43 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import 'dart:convert';
-
 import 'package:over_react_codemod/src/boilerplate_suggestors/boilerplate_utilities.dart';
 import 'package:over_react_codemod/src/boilerplate_suggestors/simple_props_and_state_class_migrator.dart';
 import 'package:test/test.dart';
 
 import '../util.dart';
-import 'boilerplate_utilities_test.dart';
 
-main() {
+void main() {
   group('SimplePropsAndStateClassMigrator', () {
-    final converter = ClassToMixinConverter();
-    final testSuggestor =
-        getSuggestorTester(SimplePropsAndStateClassMigrator(converter));
+    SimplePropsAndStateClassMigratorTestHelper();
 
-    setUpAll(() {
-      semverHelper = SemverHelper(jsonDecode(reportJson));
+    group('with --treat-all-components-as-private flag', () {
+      SimplePropsAndStateClassMigratorTestHelper(
+        shouldTreatAllComponentsAsPrivate: true,
+      );
     });
 
+    group('with invalid file path', () {
+      SimplePropsAndStateClassMigratorTestHelper(
+        path: 'test/boilerplate_suggestors/does_not_exist.json',
+        isValidFilePath: false,
+      );
+    });
+  });
+}
+
+void SimplePropsAndStateClassMigratorTestHelper({
```

Comment:
Only classes should be `UpperCamelCase`

```suggestion
void simplePropsAndStateClassMigratorTestHelper({
```